publications-2017.bib

```diff
@@ -1,5 +1,102 @@
 % Encoding: US-ASCII
 
+@PhDThesis{Cox2017,
+  Title                    = {{Adaptive large-scale mantle convection simulations}},
+  Author                   = {Cox, SP},
+  Year                     = {2017},
+  School = {University of Leicester},
+  Url                      = {https://lra.le.ac.uk/handle/2381/39571}
+}
+
+@Article{Ganesan2017,
+  Title                    = {{Microstructural Response of Magnesium Alloys: 3D Crystal Plasticity and Experimental Validation}},
+  Author                   = {Ganesan, S},
+  Year                     = {2017},
+  School = {University of Michigan},
+  Url                      = {http://www-personal.umich.edu/~veeras/papers/sriramthesis.pdf}
+}
+
+@PhdThesis{Giuliani2017,
+  Title                    = {{Modelling Fluid Structure Interaction problems using Boundary Element Method}},
+  Author                   = {Giuliani, N},
+  School                   = {Scuola Internazionale Superiore di Studi Avanzati - Trieste},
+  Year                     = {2017},
+  Url                      = {https://iris.sissa.it/handle/20.500.11767/57293}
+}
+
+@InProceedings{Goluchova,
+  Title                    = {Epicyclic oscillations of thick relativistic disks},
+  Author                   = {Goluchov{\'{a}}, K and T{\"{o}}r{\"{o}}k, G},
```

Comment:
Is this the same as http://adsabs.harvard.edu/abs/2017bhns.work...47H ? The Harvard link has a lot more authors.

Reply:
Ah, good eye! I had looked at the linked to work, and missed the additional authors. Lesson: Don't ever trust @tjhei's original list of entries.